crates/biome_analyze/src/lib.rs

```diff
@@ -406,10 +408,24 @@ where
                     return true;
                 }
 
-                suppression
+                if suppression
                     .suppressed_rules
                     .iter()
                     .any(|filter| *filter == entry.rule)
+                {
+                    return true;
+                }
+
+                if entry.instances.is_empty() {
+                    return false;
+                }
```

Comment:
Is this needed? Does line 423 `entry.instances.iter().all()` return true or false, if array is empty?

Reply:
I think `all()` returns `true` on an empty list.